topics/microbiome/tutorials/multivariable-association/tutorial.md

```diff
@@ -0,0 +1,348 @@
+>
+>    As a default, Galaxy uses the link as the name of the new dataset. It also does not link the dataset to a database or a reference genome.
+>
+>    {% snippet faqs/galaxy/datasets_rename.md %}
+>
+> 4. Inspect the content of a file.
+>
+>    > <tip-title>Inspecting the content of a dataset</tip-title>
+>    >
+>    > * Click on the {% icon galaxy-eye %} (eye) icon next to the relevant history entry
+>    > * View the content of the file in the central panel
+>    {: .tip}
+>
+>    > <question-title></question-title>
+>    >
+>    > 1.  What is the main difference between the two files?
+>    >
+>    >
+>    > > <solution-title></solution-title>
+>    > > 1. The metadata file describes sample characteristics (e.g., clinical data, demographics) while the taxonomy or feature file contains microbial data (e.g., taxa abundance) used for analysis in microbiome studies.
+>    > {: .solution }
+>    {: .question}
+>
+{: .hands_on}
+
+
+# Find associations between the two files
+Now we will find significant associations between microbial features( taxonomy file) and metadata variables (metadata file) using the **MaAslin2** tool
+
+> <hands-on-title> Task description </hands-on-title>
+>
+> 1. {% tool [MaAsLin 2](toolshed.g2.bx.psu.edu/repos/iuc/maaslin2/maaslin2/1.16.0+galaxy0) %} with the following parameters:
+>    - *"Interactions: Fixed effects"*: `c3:age`, `c4:diagnosis`
+>    - *"Random effects"*: `c5:subject`
+>    - *"Reference"*: `diagnosis,CD`
+>
+>  Keep the rest of the default values as it is.
+{: .hands_on}
+
+# Understanding parameters in the tool
+
+Let's now understand the role of each parameter in the tool.
+
+1. **Interactions: Fixed effects**: Fixed effects are the factors in your model that you want to study and draw conclusions about. These are the variables you hypothesize have a direct and consistent influence on the outcome. For example, you are studying how different diets affect gut microbiome composition, then diet would be a fixed effect because you’re specifically interested in understanding how different diets influence the microbiome. You might also include other fixed effects like age and gender to control for their impact.
+
+3. **Random effects**: In some studies, like those following people over time or studying families, samples from the same group can be similar. MaAsLin2 helps handle this by letting researchers choose a grouping factor. This helps make sure the statistical analysis is more accurate. For example, setting random_effects = "Subject_ID" helps control for the correlation between samples that come from the same individual.
+
+4. **Reference**: It allows researchers to establish a baseline or standard category against which other categories are compared, helping to interpret and understand the effects of different variables on microbial features. 
+
+   > <comment-title></comment-title>
+   > - In MaAslin2, the reference level is must for variables with more than two distinct kinds of values.
```

Comment:
Something is wrong with this sentence.

Reply:
![Screenshot from 2024-09-29 15-17-03](https://github.com/user-attachments/assets/a1fbaac2-1203-45f0-8a4a-bc02975f75a1)

@bgruening I was attempting to rephrase the clarification provided in this. Did you find any factual error in this or you want me to rephrase it more clearly? 
https://forum.biobakery.org/t/passing-multiple-reference-factors-in-maaslin2-version-1-16-0/7035/5

Reply:
"is must for" sounds just wrong to me.

Reply:
@bgruening , how about this -"In MaAslin2, the reference level is required for variables with more than two levels" . 

User will get an error if he choose a fixed effect with more than two level and forgets to choose a reference among those levels. so this is really importmant

Reply:
Yes way better.